x-pack/plugins/security_solution/public/detections/components/alerts_table/translations.ts

```diff
@@ -285,3 +285,17 @@ export const TRIGGERED = i18n.translate(
     defaultMessage: 'Triggered',
   }
 );
+
+export const ADD_TO_EXISTING_CASE = i18n.translate(
```

Comment:
Duplicated translations can be moved to cases when the logic of the cases toasters is moved inside cases.

Reply:
Yes.

Reply:
This is actually the buttons themselves not the toasts. so this should stay here.